client/components/deposits-overview/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Card, CardHeader } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import { useAllDepositsOverviews } from 'wcpay/data';
+
+interface OverviewProps {
+	overview: AccountOverview.Overview;
+	account: AccountOverview.Account;
+	isLoading: boolean;
+}
+
+/**
+ * Renders a deposits overview
+ *
+ * @param {AccountOverview.Overview} props Deposits overview
+ * @return {JSX.Element} Rendered element with deposits overview
+ */
+const DepositsOverviewDetails: React.FunctionComponent< OverviewProps > = (
+	props
+) => {
+	return (
+		<Card>
+			<CardHeader>
+				{ __( 'Deposits', 'woocommerce-payments' ) }
+			</CardHeader>
+
+			<p>Next Deposits Section Goes here</p>
+
+			<p>Deposits History Section Goes here</p>
+
+			<p>Deposits Card Footer/Action Goes here</p>
+		</Card>
+	);
+};
+
+const DepositsOverview = (): JSX.Element => {
+	const {
+		overviews,
+		isLoading,
+	} = useAllDepositsOverviews() as AccountOverview.OverviewsResponse;
+
+	const { currencies, account } = overviews;
+
+	return (
+		<React.Fragment>
+			{ currencies.map( ( overview: AccountOverview.Overview ) => (
```

Comment:
This will render a `<DepositsOverviewDetails>` card for each currency, rather than just one for the "selected" currency.

This is probably not a problem for this skeleton component issue and can be addressed later.

Reply:
Thanks for picking this up @Jinksi! For now, I've updated this to just get the first current from the list instead of looping over the array.

https://github.com/Automattic/woocommerce-payments/pull/5759/commits/6c577b866472d1da49db12ca8db262503d6ceace

We will have to figure out how we'll get the current currency that has been selected on the Account Balances, but that can come later I think.